apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx

```diff
@@ -8,6 +8,7 @@ export default class SingleChoiceResponses extends React.Component {
     question: PropTypes.string.isRequired,
```

Comment:
should we change the name of this component now that it isn't just single-choice responses?